src/FSharpx.Collections/Collections.fs

```diff
@@ -481,8 +493,35 @@ module Dictionary =
         | true,v -> Some v
         | _ -> None
 
+    let inline toSeq (d:seq<KeyValuePair<'a, 'b>>) : seq<'a * 'b> =
+        d |> Seq.map (fun (KeyValue(x,y)) -> x, y)
+
+
+[<Extension>]
+type ExtraIDictionary() =
```

Comment:
What are the types ExtraIDictionary and ExtraIReadOnlyDictionary ?

Reply:
Just a class to hold the extension methods.